plugins/scanners/scanoss/src/test/kotlin/ScanOssResultParserTest.kt

```diff
@@ -125,11 +162,65 @@ class ScanOssResultParserTest : WordSpec({
                                 "."
                             ),
                             "pkg:github/vdurmont/semver4j",
-                            SpdxExpression.parse("CC-BY-SA-2.0")
+                            SpdxExpression.parse("CC-BY-SA-2.0"),
+                            mapOf(
+                                "all_purls" to "pkg:github/vdurmont/semver4j"
+                            )
                         )
                     )
                 )
             )
         }
+
+        "should handle multiple PURLs by selecting first as primary and preserving all in metadata" {
+            val results = File("src/test/assets/scanoss-multiple-purls.json").readText().let {
+                JsonUtils.toScanFileResultsFromObject(JsonUtils.toJsonObject(it))
+            }
```

Comment:
By now, tests assets have been moved to resources in `main`, so you need to rebase and adapt this code as show e.g. [here](https://github.com/oss-review-toolkit/ort/pull/10318/files#diff-abf375b12f262cb8f7df2d3b00b86e7b8d860c2bc25abec91c73f61b7ee243faR47).